src/components/common/contact-form.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import ContactFormStyles from './contact-form.module.scss'
+
+export default ({ messageCopy }) => {
+  return (
+    <form
+      className={ContactFormStyles.contactForm}
+      name="contact"
+      method="POST"
+      data-netlify-recaptcha="true"
+      data-netlify="true"
+    >
+      <div>
```

Comment:
Netlify also requires a hidden input element with Gatsby, which I just learned and isn't documented anywhere. Let's add:

```
<input type="hidden" name="form-name" value={name} />`